README.md

```diff
@@ -5,11 +5,11 @@ Send an SMS from GitHub Actions.
 ## Prerequisites
 
 - A Twilio Account. [Sign up for free](https://www.twilio.com/try-twilio)
-- A [Twilio API Key and Secret](https://www.twilio.com/docs/iam/keys/api-key)
+- A [Twilio Auth Token](https://www.twilio.com/docs/iam/api/authtoken)
```

Comment:
Might want to include a strong recommendation for a verified number

Reply:
Linked to regulatory requirement FAQ